src/cljs/ui/component/app_container.cljs

```diff
@@ -7,73 +7,73 @@
             [reagent-material-ui.core.menu :refer [menu]]
             [reagent-material-ui.core.menu-item :refer [menu-item]]
             [reagent-material-ui.core.typography :refer [typography]]
-            [reagent.core :as reagent]
             [shared.components :refer [button collapsible-tab spread-logo]]
             [ui.component.icon :refer [arg->icon icons]]
             [ui.component.search :refer [search-bar]]
             [ui.format :refer [format-percentage]]
             [ui.router.subs :as router.subs]
             [ui.subscriptions :as subs]
-            [ui.utils :as ui-utils :refer [>evt dispatch-n]]))
+            [ui.utils :as ui-utils :refer [>evt dispatch-n type->tab]]))
 
 (def type->label {"CONTINUOUS_TREE"       "Continuous: MCC tree"
                   "DISCRETE_TREE"         "Discrete: MCC tree"
                   "BAYES_FACTOR_ANALYSIS" "Discrete: Bayes Factor Rates"})
 
-
-(def type->tab {"CONTINUOUS_TREE"       "continuous-mcc-tree"
-                "DISCRETE_TREE"         "discrete-mcc-tree"
-                "BAYES_FACTOR_ANALYSIS" "discrete-rates"})
-
 (defn completed-menu-item [_]
+  (fn [{:keys [id readable-name of-type status new?]}]
+    (let [badge-text                       (cond
+                                             (= status "ERROR") "Error"
+                                             new?               "New")
+          [anchorElement setAnchorElement] (react/useState nil)
```

Comment:
why are we using react hooks instead of just reagent/atoms or re-frame/db for state? I don't think is a good idea to have so many redundant state management functionality. Or am I missing something?

Reply:
it was cargo culted - fixed it here and in user component